packages/node-http-handler/src/node-http-handler.ts

```diff
@@ -83,7 +83,13 @@ export class NodeHttpHandler implements HttpHandler {
         resolve({ response: httpResponse });
       });
 
-      req.on("error", reject);
+      req.on("error", (err: Error) => {
+        if (["ECONNRESET", "EPIPE", "ETIMEDOUT"].includes((err as any).code)) {
```

Comment:
Can we pull these out into a constant somewhere?

Reply:
Updated in [`3b0d3d0` (#1693)](https://github.com/aws/aws-sdk-js-v3/pull/1693/commits/3b0d3d03141d5ab4c1614beb174b58d49f1232e8)